pkgs/development/python-modules/chex/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, absl-py
+, dm-tree
+, jax
+, jaxlib
+, numpy
+, toolz
+}:
+
+buildPythonPackage rec {
+  pname = "chex";
+  # As of 2021-12-29, the latest official version has broken tests with jax 0.2.26.
+  version = "0.1.0+date=2021-12-16";
```

current nixpkgs convention for choosing specific 
```suggestion
  version = "unstable-2021-12-16";
```

---

Done.